examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -49,11 +49,16 @@ async function start(): Promise<void> {
     const consoleLogger: ConsoleLogger = new ConsoleLogger();
 
     // Get or create the document depending if we are running through the create new flow
+
     const client = useFrs ? FrsClient :  new TinyliciousClient();
-    const [fluidContainer, containerServices] = createNew
+    const containerAndServices = createNew
         ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
         : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
 
+    // [fluidContainer, containerServices]
+    const { fluidContainer,containerServices } = { fluidContainer: containerAndServices.fluidContainer,
```

Comment:
Nit: spacing, also we can remove the comment and rename to resources
```suggestion
    const { fluidContainer, containerServices } = containerAndServices
```

Reply:
Renaming containerAndServices to clientResources. Does the variable name make sense?😅